package.json

```diff
@@ -88,5 +90,16 @@
   },
   "files": [
     "dist"
-  ]
-}
+  ],
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged",
+      "pre-push": "npm test"
+    }
+  },
+  "lint-staged": {
+    "*.{js,ts}": [
```

Comment:
Is there a reason we have to define this? Why can't it be part of the `npm run lint` script? Why do we need to define it in two places?

Reply:
I tried and it will also check package.json for instance.  Lint-staged supplies files to be checked and it will override `--ext .ts,.js `parameter for eslint.